gRPC-ProtoRPC.podspec

```diff
@@ -28,6 +28,10 @@ Pod::Spec.new do |s|
   s.license  = 'Apache License, Version 2.0'
   s.authors  = { 'The gRPC contributors' => 'grpc-packages@google.com' }
 
+  # gRPC podspecs depend on fix for https://github.com/CocoaPods/CocoaPods/issues/6024,
+  # which was released in Cocoapods v1.2.0.
+  s.cocoapods_version = '>= 1.2.0'
```

Comment:
Given that the fix is needed by `gRPC-Core.podspec`, why not put the restriction just there, and let dependent podspecs just inherit it?